docker-app/qfieldcloud/core/raw_query_containers.py

```diff
@@ -0,0 +1,5 @@
+from collections import namedtuple
+
+UserEmailDetails = namedtuple(
+    "UserEmailDetails", ["username", "first_name", "last_name", "email"]
+)
```

Comment:
I would suggest we move this to the main `admin.py` file, as it is easier to make changes in the future. (we will need to add few columns to the query)